javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/typesolvers/CombinedTypeSolver.java

```diff
@@ -34,13 +35,29 @@
 
     private TypeSolver parent;
     private List<TypeSolver> elements = new ArrayList<>();
+    private Predicate<Exception> errorFilter;
 
     public CombinedTypeSolver(TypeSolver... elements) {
+        this(ExceptionFilters.IGNORE_ALL, elements);
+    }
+
+    /** @see #setFilter(Predicate) */
+    public CombinedTypeSolver(Predicate<Exception> errorFilter, TypeSolver... elements) {
+        setFilter(errorFilter);
+
         for (TypeSolver el : elements) {
             add(el);
         }
     }
 
+    /**
+     * @param errorFilter A filter which determines if an exception raised while solving should be ignored.
+     *            Should return <code>true</code> when the exception must be <b>ignored</b>.
+     */
+    public void setFilter(Predicate<Exception> errorFilter) {
```

Comment:
I would call it "ExceptionHandler" instead of Filter

Reply:
We could explain better the use case because it is far from being obvious

Reply:
Will do.